launchable/commands/record/session.py

```diff
@@ -38,12 +47,18 @@ def session(build_name: str, save_session_file: bool, print_session: bool = True
         "Content-Type": "application/json",
     }
 
-    client = LaunchableClient(token)
+    flavor_dict = {}
+    for f in flavor:
+        flavor_dict[f[0]] = f[1]
```

Comment:
Extra credit: flag and reject duplicate keys since that'd be an user error.

Reply:
yes indeed, so I'll add todo comment and merge this branch